lib/ui/volatile_path_tracker.h

```diff
@@ -66,14 +67,10 @@ class VolatilePathTracker {
 
  private:
   fml::RefPtr<fml::TaskRunner> ui_task_runner_;
```

Comment:
This is now _only_ used for debug checking purposes. I don't know if we should just drop it...